### ISSUE TITLE
Wrong dependency - itsdangerous 1.0.0

### ISSUE THREAD

**Issue by shevelevs** [pypa/warehouse#4943]
**Describe the bug**
make build fails because of dependency on itsdangerous=1.0.0 which seems to be deleted

**Expected behavior**
There should be no dependency on the missing package

**To Reproduce**
run make build -->

```Could not find a version that satisfies the requirement itsdangerous==1.0.0 (from -r /tmp/requirements/main.txt (line 238)) (from versions: 0.9, 0.9.1, 0.10, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 1.1.0)
No matching distribution found for itsdangerous==1.0.0 (from -r /tmp/requirements/main.txt (line 238))
```

**My Platform**
mac os

**Comment by shevelevs** [comment]
the change to requirements/main.txt that seems to fix the build (haven't run the tests though)
```
-itsdangerous==1.0.0 \
-    --hash=sha256:a7de3201740a857380421ef286166134e10fe58846bcefbc9d6424a69a0b99ec \
-    --hash=sha256:aca4fc561b7671115a2156f625f2eaa5e0e3527e0adf2870340e7968c0a81f85
+itsdangerous==1.1.0 \
+    --hash=sha256:b12271b2047cb23eeb98c8b5622e2e5c5e9abd9784a153e9d8ef9cb4dd09d749 \
+    --hash=sha256:321b033d07f2a4136d3ec762eac9f16a10ccd60f53c0c91af90217ace7ba1f19
```

**Comment by xni** [comment]
It is already fixed by https://github.com/pypa/warehouse/issues/4939

**Comment by di** [comment]
Thanks for the report! This was fixed in #4940.
